src/pybricks/_common.py

```diff
@@ -1285,3 +1285,77 @@ def hsv(self, surface: bool = True) -> Color:
             Measured color. The color is described by a hue (0--359), a
             saturation (0--100), and a brightness value (0--100).
         """
+
+
+class BLE:
+    """
+    Bluetooth Low Energy.
+
+    .. versionadded:: 3.3
+    """
+
+    def broadcast(self, *args: Union[None, bool, int, float, str, bytes]) -> None:
+        """broadcast([arg1][, arg2][...])
+
+        Starts broadcasting advertising data containing *args*.
```

Comment:
Starts broadcasting the given data values. Each value can be an integer, a floating point value,  a string, bytes, ``None``, ``True``, or ``False``. The data is broadcasted on the `broadcast_channel` you selected when initializing the hub.

The total data size is quite limited. For example, ....